Description: This patch adds string literals
 to format strings in the snprintf function. This is required by
 -Werror=format-security to avoid format string attacks. This pacth
 was backported from siege beta in order to be able to use security
 hardening build flags.
Origin: upstream, http://www.joedog.org/pub/siege/beta/siege-beta.tar.gz
Last-Update: 2012-10-07
--- a/src/init.c
+++ b/src/init.c
@@ -48,7 +48,7 @@
    * use default of ~/.siegerc */
   if(strcmp(my.rc, "") == 0){
     if((e = getenv("SIEGERC")) != NULL){
-      snprintf(my.rc, sizeof(my.rc), e);
+      snprintf(my.rc, sizeof(my.rc), "%s", e);
     } else {
       snprintf(my.rc, sizeof(my.rc), "%s/.siegerc", getenv("HOME"));
       if (stat(my.rc, &buf) < 0 && errno == ENOENT) {
